code/datums/supply_packs/cargo.dm

```diff
@@ -0,0 +1,111 @@
+	name = "Tractor"
+	contains = list(/obj/structure/bed/chair/vehicle/tractor,
+					/obj/machinery/cart/cargo)
+	cost = 40
+	containertype = /obj/structure/largecrate
+	containername = "tractor crate"
+	group = "Cargo"
+	containsicon = /obj/structure/bed/chair/vehicle/tractor
+	containsdesc = "The classic ride for any cargo man. Comes with one tractor, and as a limited time bonus: a free cart!"
+
+/datum/supply_packs/carts
+	name = "Carts"
+	contains = list(/obj/machinery/cart/cargo,
+                    /obj/machinery/cart/cargo)
+	cost = 15
+	containertype = /obj/structure/largecrate
+	containername = "carts crate"
+	group = "Cargo"
+	containsdesc = "Two carts, usable with any tractor. Tractor not included."
+
+/datum/supply_packs/mining
+	name = "Mining equipment"
+	contains = list(/obj/item/weapon/pickaxe/drill,
+					/obj/item/weapon/pickaxe,
+					/obj/item/weapon/pickaxe,
+					/obj/item/device/flashlight/lantern,
+					/obj/item/device/flashlight/lantern,
+					/obj/item/device/flashlight/lantern,
+					/obj/item/device/mining_scanner,
+					/obj/item/weapon/storage/bag/ore,
+					/obj/item/weapon/storage/bag/ore,
+					/obj/item/weapon/storage/bag/ore,
+					/obj/item/weapon/storage/bag/money,
+					/obj/item/weapon/storage/bag/money)
+	cost = 20
+	containertype = /obj/structure/closet/crate/basic
+	containername = "mining equipment crate"
+	access = list(access_mining)
+	group = "Cargo"
+	containsdesc = "An emergency set of basic mining gear. Comes with enough gear for three miners to do their jobs. One gets a fancy drill, but the other two get entire bags to put their money in."
+
+/datum/supply_packs/rcs_device
+	name = "Rapid-Crate-Sender"
+	contains = list (/obj/item/weapon/rcs)
+	cost = 80
+	containertype = /obj/structure/closet/crate/engi
+	containername = "\improper RCS crate"
+	group = "Cargo"
+	containsdesc = "A rapid crate sending device. Comes pre-loaded with a rechargeable cell. Telepads not included!"
+
+/datum/supply_packs/rcs_telepad
+	name = "Cargo telepads"
+	contains = list (/obj/item/device/telepad_beacon,
+					 /obj/item/device/telepad_beacon,
+					 /obj/item/device/telepad_beacon)
+	cost = 80
+	containertype = /obj/structure/closet/crate/engi
+	containername = "\improper RCS telepad crate"
+	group = "Cargo"
+	containsdesc = "Telepads for use with the rapid crate sender. Contains three."
+
+/datum/supply_packs/automation
+	name = "Automation supplies"
+	contains = list(/obj/item/weapon/circuitboard/autoprocessor/wrapping,
+					/obj/item/weapon/circuitboard/autoprocessor/clothing,
+					/obj/item/weapon/circuitboard/sorting_machine/item,
+					/obj/item/weapon/circuitboard/crate_opener,
+					/obj/item/weapon/circuitboard/crate_closer)
+	cost = 25
+	containertype = /obj/structure/closet/crate/engi
+	containername = "automation supplies crate"
+	group = "Cargo"
+	containsdesc = "A starter pack for automating cargo. Includes an automatic crate opener, closer, wrapper, sorter, and even a dresser."
```

Comment:
Probably change that to even an autoclother. Dresser sounds like you're getting furniture.

Reply:
It's now an auto-dresser, the worst of both worlds.